TombEditor/EditorActions.cs

```diff
@@ -3985,15 +3985,15 @@ public static bool BuildLevel(bool autoCloseWhenDone, IWin32Window owner, bool s
                 _editor.SendMessage(whatLoaded, PopupType.Info);
 
             using (var form = new FormOperationDialog("Build level", autoCloseWhenDone, false,
-                progressReporter =>
+                (progressReporter,cancelToken) =>
```

Comment:
Please fix formatting and add spaces after commas (e.g. in this case, it should be `(progressReporter, cancelToken)` where it's needed.